libbeat/scripts/Makefile

```diff
@@ -116,7 +117,8 @@ integration-tests-environment:
 .PHONY: system-tests
 system-tests: libbeat.test prepare-tests system-tests-setup
 	. build/system-tests/env/bin/activate; nosetests -w tests/system --process-timeout=$(TIMEOUT) --with-timer
-	python ../scripts/aggregate_coverage.py -o ${COVERAGE_DIR}/system.cov ./build/system-tests/run
+	python ${GOPATH}/src/github.com/elastic/beats/scripts/aggregate_coverage.py -o ${COVERAGE_DIR}/system.cov ./build/system-tests/run
```

Comment:
@andrewkroh : Thanks for your comment.

I used Godep to save the dependencies in the `vendor` folder. This tool only copy the go files. 
So, I had to use the script from my GOPATH.

glide tool copies the entire dependency folders in `vendor` folder. Then, it is best to use the script in the vendor folder.

I will set a variable in the libbeat/scripts/Makefile.
This variable will have `..` as a default value.
Community beat Makefile can override it to `./vendor/github.com/elastic/beats/`

Does it approach fit everyone?